### PR TITLE
Strategy Icons

### PR DESCRIPTION
**Pull request by HojouFotytu**
Added new strategy shortcuts for:
- Buy/Sell Formula
- Rebuy Timeout
- min/max  change perc

### CODE REVIEW

File: Core/ProfitTrailer/StrategyHelper.cs [hunk 1]
```diff
@@ -143,11 +160,8 @@ public static string GetStrategyShortcut(string strategyName, bool onlyValidStra
         case "rebuy timeout":
           result = "TIMEOUT";
           break;
-        case "min/max change perc":
```
Comment (djbadders) [inline Core/ProfitTrailer/StrategyHelper.cs]: Should this be removed? Doesn't this show up if you set a max or min percentage change?

Comment (HojouFotytu) [reply Core/ProfitTrailer/StrategyHelper.cs]: Yeah, thanks.  It didn't work there -- i think because of the slash with ToLower.

Comment (HojouFotytu) [reply Core/ProfitTrailer/StrategyHelper.cs]: I also forgot to remove the rebuy timout as well.  So sloppy...  ;)

Comment (HojouFotytu) [reply Core/ProfitTrailer/StrategyHelper.cs]: BTW, min/max is only applied if a coin exceeds the set values, and can't be bought.

**Pull request merged**